README.md

```diff
@@ -124,12 +118,44 @@ For in-depth examples of NNCF integration, browse the [sample scripts](#model-co
 For FAQ, visit this [link](./docs/FAQ.md).
 
 
+## Usage of Post-Training Quantization
+
+The basic workflow steps:
+1) Initialize the dataset transformation function.
+2) Initialize NNCF Dataset.
```

Comment:
It looks more consistent if we initialize the `Dataset` first, and then the transformation function.

Reply:
You should provide the transformation func to nncf.Dataset()

Reply:
From the user's perspective, there is no difference between `NNCF Dataset` and `nncf.Dataset()` (based on the README).
If it's different entities, describe it properly for the each step, please.

Reply:
> From the user's perspective, there is no difference between `NNCF Dataset` and `nncf.Dataset()` (based on the README). If it's different entities, describe it properly for each step, please.

from the usage example it is super clear what it means, no?